src/index.js

```diff
@@ -1,35 +1,95 @@
 'use strict';
 
+const mapping = require('ember-modules-codemod/config/mapping');
+
 module.exports = function(babel) {
-  let t = babel.types;
+  const t = babel.types;
+
+  // Flips the codemod mapping into an 'import' indexed object, that exposes the
+  // default import as well as named imports, e.g. import {foo} from 'bar'
+  const reverseMapping = {};
+  Object.keys(mapping).forEach(global => {
+    const imported = mapping[global];
+    const importRoot = imported[0];
+    let importName = imported[1];
+    if (!importName) {
+      importName = 'default';
+    }
+
+    if (!reverseMapping[importRoot]) {
+      reverseMapping[importRoot] = {};
+    }
+
+    reverseMapping[importRoot][importName] = global;
+  });
 
   return {
     name: 'ember-modules-api-polyfill',
-
     visitor: {
       ImportDeclaration(path) {
         let node = path.node;
         let replacements = [];
 
-        if (node.source.value === '@ember/component') {
-          node.specifiers.forEach(specifier => {
-            let name = specifier.local;
-
-            if (specifier.type === 'ImportDefaultSpecifier') {
-              replacements.push(
-                t.variableDeclaration('var', [
-                  t.variableDeclarator(
-                    name,
-                    t.memberExpression(t.identifier('Ember'), t.identifier('Component'))
-                  ),
-                ])
-              );
-            }
-          });
-
-          if (replacements.length > 0) {
-            path.replaceWithMultiple(replacements);
+        // Ignore non @ember imports
+        if (!importRoot.startsWith('@ember/')) {
```

Comment:
I _think_ this should be `node.source.value` (CI is saying that `importRoot` is not defined)...